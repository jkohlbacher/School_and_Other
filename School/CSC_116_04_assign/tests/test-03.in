cat
bat
its
mut
all
zap
zop
cup
zzz

