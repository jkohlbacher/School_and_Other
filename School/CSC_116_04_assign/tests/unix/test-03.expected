Read 178691 words.
Tiles: [cat]
Solution: 1 [act] Score [5]
Solution: 2 [cat] Score [5]
Tiles: [bat]
Solution: 1 [bat] Score [5]
Solution: 2 [tab] Score [5]
Tiles: [its]
Solution: 1 [its] Score [3]
Solution: 2 [sit] Score [3]
Solution: 3 [tis] Score [3]
Tiles: [mut]
Solution: 1 [mut] Score [5]
Tiles: [all]
Solution: 1 [all] Score [3]
Tiles: [zap]
Solution: 1 [zap] Score [14]
Tiles: [zop]
Solution: 1 [op] Score [4]
Tiles: [cup]
Solution: 1 [cup] Score [7]
Tiles: [zzz]
Solution: 1 [zzz] Score [30]
Tiles: []
